example/scalalib/testing/5-test-stealing/build.mill

```diff
@@ -0,0 +1,38 @@
+// Test stealing is an opt-in, powerful feature that enables parallel test execution while maintaining complete reliability and debuggability.
```

Comment:
`while maintaining complete reliability and debuggability.` is meaningless. You should write why people would turn this on: because it provides both the speedups from parallelism and the efficiency of re-using JVMs between tests, allowing it to be used on all sorts of test suites without downside. And we will likely make it the default in future